hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
`subsystems.xml` and `template.xml` are there to produce all standalone*.xml files for a runnable WF distro.

I admit I have just mechanically taken the files from WildFly feature pack [1] and enhanced them with the Agent stuff. I assumed my mechanical approach is well justified by the fact that the Agent feature pack depends on the WF feature pack [2]. You can clearly see from the blame [2] that it has been for a long time like that already. In other words, given that our feature pack depends on WF feature pack, nobody (incl. me) should assume that the agent will be able to run on top of WF Core [3] or some stripped-down version of WF.

If you believe the Agent feature pack should depend on something than the WF feature pack, please state it clearly. But in such a case, I'd vote for merging the present, WF feature pack based PR first and try to change the base after that.

[1] https://github.com/wildfly/wildfly/tree/master/feature-pack/src/main/resources/configuration/standalone
[2] https://github.com/ppalaga/hawkular-agent/blame/ac5462eaaf0255ba912fe3439052068ecd805a1e/hawkular-wildfly-agent-feature-pack/feature-pack-build.xml#L23
[3] https://github.com/wildfly/wildfly-core/tree/master/core-feature-pack